runtime/doc/version9.txt

```diff
@@ -41614,7 +41614,7 @@ Functions: ~
 Autocommands: ~
 
 |CursorMovedC|		after the cursor was moved in the comamnd-line
-|KeyInputPre|		process any Key event in any mode
+|KeyInputPre|		before the process any key event in any mode
```

Comment:
```suggestion
|KeyInputPre|		before processing any key event in any mode
```

Reply:
Thanks! "Commit Suggestion" failed because the file size was too large, so I fixed it locally.